woocommerce/woocommerce-gateway-stripe
# Keep the enabled payment methods for UPE in the config when disabling the UPE gateway

# Changes proposed in this Pull Request:

Issue: #1998
Fixes #1998

This PR keeps the enabled payment methods for UPE in the config when disabling the UPE gateway.

# Testing instructions

- Enable UPE
- Click Save
- Enable only Card, SEPA, and Bancontact
- Click Save
- Disable UPE
- Click Save
- Verify that Card, SEPA, and Bancontact are still enabled without UPE
- Disable Bancontact
- Enabled PE
- Save
- Verify that only Card and SEPA are enabled
